example/src/main/java/org/wordpress/android/fluxc/example/utils/RandomStringUtils.kt

```diff
@@ -0,0 +1,44 @@
+package org.wordpress.android.fluxc.example.utils
+
+object RandomStringUtils {
+    @JvmStatic
+    fun randomAlphanumeric(count: Int): String {
+        return randomString(
+            count = count,
+            letters = true,
+            numbers = true
+        )
+    }
+
+    @JvmStatic
+    fun randomAlphabetic(count: Int): String {
+        return randomString(
+            count = count,
+            letters = true,
+            numbers = false
+        )
+    }
+
+    @JvmStatic
+    fun randomNumeric(count: Int): String {
+        return randomString(
+            count = count,
+            letters = false,
+            numbers = true
+        )
+    }
+
+    private fun randomString(count: Int, letters: Boolean, numbers: Boolean): String {
```

Comment:
This is a personal preference, but I really like having default values for `letters` & `numbers`, so only the specific parameters would need to be configured.

```suggestion
    private fun randomString(count: Int, letters: Boolean = true, numbers: Boolean = true): String {
```

Reply:
I was thinking about that as well, but I ended up not using default values for `letters` & `numbers` as I liked the individual functions being explicit about that, it seemed more readable to me, that is, in terms of making sense of what is happening without needing to look at the private `randomString(...)` function.

This is now done: b1a06eaacf0ee09280306f0a219d698cc34e744b

Let me know how you like your suggested alternative @oguzkocer ! 🙏